frontend/src/components/maps/leaflet/Map.scss

```diff
@@ -13,3 +13,18 @@
     margin: 0px;
   }
 }
+
+.leaflet-control-layers {
+  &.leaflet-control-layers-expanded {
+    border-radius: 0px;
+  }
+  .leaflet-control-layers-overlays {
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    width: 150px;
+    label {
+      display: flex !important;
```

Comment:
Using `!important` generally means we're doing something wrong.

Reply:
Thats an ooops, removed it